src/plugins/interval-trainer/interval-trainer-display.js

```diff
@@ -1,279 +0,0 @@
-import { Button } from 'antd';
-import { Piano } from 'react-piano';
-import { RadialChart } from 'react-vis';
```

Comment:
Can this guy be uninstalled then?

Reply:
Good catch, I didn't think about the modules. I could remove both `react-piano` and `react-vis` :man_dancing: